FileIO/XmlIO/Boost/BoostXmlGmlInterface.h

```diff
@@ -68,6 +68,9 @@ class BoostXmlGmlInterface : public XMLInterface
 	                  const std::vector<std::size_t>& pnt_id_map,
 	                  std::map<std::string, std::size_t>*& sfc_names);
 
+	void addSurfacesToPropertyTree(boost::property_tree::ptree & pt);
+	void addPolylinesToPropertyTree(boost::property_tree::ptree & geometry_set);
```

Comment:
Since the method names are self-explanatory I omitted the documentation. Do you think documentation is necessary?